enhancements/declarative-index-config.md

```diff
@@ -865,34 +865,22 @@ $ tree
 ├── etcd.json
 ├── amqstreams.json
 ```
-#### Editing a package inside an index
+#### Creating new indexes using the config files/Editing a package inside an existing index
 
-A new sub-command `update` will be introduced under `opm index`, which will take a folder containing json representations of packages as it's input along with an existing index image tag, and will replace the edited config files in the index container. 
+A new sub-command `build` will be introduced under `opm index`, which will take a folder containing json representations of packages as it's input, and will build a container image with the folder and the `opm` binary in it. 
```

Comment:
>We may even want a more generic description in case we define a new declarative syntax at some point.

Is an enhancement proposal supposed to be a "live" document like a design doc in our documentation site? Or is it supposed to only account for the current state of the world, and an outline of the next state of the world after the proposal is accepted?